pkg/drivers/kvm/numa.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2021 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kvm
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"text/template"
+)
+
+// NUMATmpl NUMA XML Template
+const NUMATmpl = `
+<numa>
+  {{- range $idx,$val :=. }}
+  <cell id='{{$idx}}' cpus='{{$val.CPUTopology}}' memory='{{$val.Memory}}' unit='MiB'/>
+  {{- end }}
+</numa>
+`
+
+// NUMA this struct use for NUMATmpl
+type NUMA struct {
+	CPUCount    int
+	Memory      int
+	CPUTopology string
+}
+
+// GetNUMAXml generate numa xml
+// evenly distributed cpu core & memory to each numa node
+func GetNUMAXml(cpu, memory, numaCount int) (string, error) {
```

Comment:
It looks like `GetNUMAXml` doesn't need to be public. Also probably it's better to follow Golang code [conventions](https://golang.org/doc/effective_go#Getters) and not use `Get` prefix in function names

Reply:
resloved @ilya-zuyev